db/migrate/20231017180938_create_tasks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.text :description
+      t.boolean :completed, default: false
+      t.references :pet, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
We have a couple checklist tables I think we were intending to reuse here instead of creating a new table. `checklist_template_items` and `checklist_template`.

Reply:
copy that